pkgs/top-level/aliases.nix

```diff
@@ -237,6 +237,7 @@ mapAliases ({
   dart_stable = dart; # Added 2020-01-15
   dart-sass-embedded = throw "dart-sass-embedded has been removed from nixpkgs, as is now included in Dart Sass itself.";
   dat = nodePackages.dat;
+  dbeaver = throw "'dbeaver' has been removed, as it was broken and outdated"; # Added 2024-05-15
```

Comment:
```suggestion
  dbeaver = dbeaver-bin; # Added 2024-05-15
```

Reply:
Glad to hear it! I've amended the message to indicate the replacement.

I'm not keen on (silently) redirecting to the new `-bin` package, since it's not compiled from source, and thus may be undesirable for some people, for one reason or another.